mountpoint-s3/tests/fuse_tests/write_test.rs

```diff
@@ -336,7 +327,9 @@ where
         filesystem_config,
         ..Default::default()
     };
-    let (mount_point, _session, _test_client) = creator_fn("fstat_after_writing", session_config);
+    let test_session = creator_fn("fstat_after_writing", session_config);
+    let mount_point = test_session.mount_dir;
+    let _test_client = test_session.test_client;
```

Comment:
I believe we can drop this.

```suggestion
```

If so, can we apply it everywhere for unused `_test_client`?

Reply:
Good catch! Fixed with https://github.com/awslabs/mountpoint-s3/pull/1096/commits/8506e33761d9cd56d982401c4e0ae6326e9574b1